### DISCUSSION TITLE
vite有没有动态加载、模块共享的解决方案

### DISCUSSION THREAD

**Discussion by tky753** [vbenjs/vue-vben-admin#249] [General]
想要实现一个高度可扩展的项目，包括一个**基座项目**和很多很多的**页面模块**。
**页面模块**独立开发，单独编译，但依赖**基座项目**，编译后放在**基座项目**的**plugin**目录下，
**基座项目**跳转页面的时候调用`http://基座项目/plugin/pageA/remoteEntry.js`加载该模块。
是的，就是**webpack5**的**模块邦联**，不知道**vite**+**rollup**有没有类似的解决方案.

**Comment by anncwb** [comment]
目前我没去仔细看这方面的,不过现在应该还不支持。你可以到vite的Discussions提下你的问题，或许那边可以帮助到你.

**Reply by 2yllll** [reply]
微前端解决方案-qiankun，我司在用，我负责的，用了蛮久了，没太大问题。遇到问题可咨询我
